loginsightwebhookdemo/bugzilla.py

```diff
@@ -27,13 +27,12 @@
 
 # Route without <ALERTID> are for LI, with are for vROps
 @app.route("/endpoint/bugzilla", methods=['POST'])
-@app.route("/endpoint/bugzilla/<ALERTID>", methods=['PUT'])
 @app.route("/endpoint/bugzilla/<TOKEN>", methods=['POST'])
-@app.route("/endpoint/bugzilla/<TOKEN>/<ALERTID>", methods=['PUT'])
+@app.route("/endpoint/bugzilla/<TOKEN>/<ALERTID>", methods=['POST','PUT'])
 @app.route("/endpoint/bugzilla/<TOKEN>/<PRODUCT>/<COMPONENT>/<VERSION>", methods=['POST'])
-@app.route("/endpoint/bugzilla/<TOKEN>/<PRODUCT>/<COMPONENT>/<VERSION>/<ALERTID>", methods=['PUT'])
-@app.route("/endpoint/bugzilla/<PRODUCT>/<COMPONENT>/<VERSION>", methods=['POST'])
-@app.route("/endpoint/bugzilla/<PRODUCT>/<COMPONENT>/<VERSION>/<ALERTID>", methods=['PUT'])
+@app.route("/endpoint/bugzilla/<TOKEN>/<PRODUCT>/<COMPONENT>/<VERSION>/<ALERTID>", methods=['POST','PUT'])
+@app.route("/endpoint/bugzilla2/<PRODUCT>/<COMPONENT>/<VERSION>", methods=['POST'])
```

Comment:
You cannot have two routes with the same methods and the same number of parameters. Since vROps requires both POST and PUT, we now have `/endpoint/bugzilla/<TOKEN>/<PRODUCT>/<COMPONENT>/<VERSION>` and `/endpoint/bugzilla2/<PRODUCT>/<COMPONENT>/<VERSION>/<ALERTID>`, which both have four parameters.